Bottom: 64de0fcfbbf746d5fd9b1d0137de49d1a3ee8f1b
Top:    048f900b4aad84c1acd02eaa9a50f9ddb8d17572
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-15 02:41:21 +0000

Refresh of butcher-the-xor-crosshair-draw

---

diff --git a/src/crosshair.c b/src/crosshair.c
index 283da2b..b809d6b 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -303,7 +303,7 @@ DrawAttached (DrawAPI *dapi)
   dapi->gc = dapi->graphics->make_gc ();
 
   dapi->graphics->set_color (dapi->gc, Settings.CrosshairColor);
-  dapi->graphics->set_draw_xor (dapi->gc, 1);
+//  dapi->graphics->set_draw_xor (dapi->gc, 1);
   dapi->graphics->set_line_cap (dapi->gc, Trace_Cap);
   dapi->graphics->set_line_width (dapi->gc, 1);
 
@@ -470,7 +470,7 @@ DrawMark (DrawAPI *dapi)
   dapi->gc = dapi->graphics->make_gc ();
 
   dapi->graphics->set_color (dapi->gc, Settings.CrosshairColor);
-  dapi->graphics->set_draw_xor (dapi->gc, 1);
+//  dapi->graphics->set_draw_xor (dapi->gc, 1);
   dapi->graphics->set_line_cap (dapi->gc, Trace_Cap);
   dapi->graphics->set_line_width (dapi->gc, 1);
